docs/src/main/asciidoc/cdi-reference.adoc

```diff
@@ -136,7 +136,7 @@ quarkus.arc.exclude-dependency.acme.artifact-id=acme-services <2>
 == Native Executables and Private Members
 
 Quarkus is using GraalVM to build a native executable.
-One of the limitations of GraalVM is the usage of https://github.com/oracle/graal/blob/master/docs/reference-manual/native-image/Limitations.md#reflection[Reflection, window="_blank"].
+One of the limitations of GraalVM is the usage of https://www.graalvm.org/21.3/reference-manual/native-image/Reflection/[Reflection, window="_blank"].
```

Comment:
Maybe let's point to the latest version that we will soon use: https://www.graalvm.org/22.2/reference-manual/native-image/dynamic-features/Reflection/ .
I don't think we need to upgrade for every version as it's about concepts but better start with the latest.

Reply:
Ok, done.